app/src/main/res/layout-sw600dp-land/hints_summary.xml

```diff
@@ -91,18 +89,17 @@
       android:layout_height="wrap_content"
       android:layout_marginStart="176dp"
       android:layout_marginEnd="176dp"
+      android:layout_marginTop="108dp"
       android:layout_marginBottom="32dp"
+      app:layout_constraintTop_toTopOf="parent"
       android:orientation="vertical"
       android:visibility="@{isListExpanded? View.VISIBLE : View.GONE}">
 
       <TextView
         android:id="@+id/hints_and_solution_summary"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
         android:layout_gravity="start"
-        android:fontFamily="sans-serif"
         android:textColor="@color/oppiaPrimaryText"
-        android:textSize="16sp" />
+        style="@style/Body" />
     </LinearLayout>
-  </LinearLayout>
-</layout>
+  </androidx.constraintlayout.widget.ConstraintLayout>
+</layout>
```

Comment:
EOF is missing, add one blank line at the end.

Reply:
Added an extra line here. File now ends at line 106